README.md

```diff
@@ -59,6 +59,7 @@ All options are optional, none are required.
 | ---- | ----------- | ------- |
 | **root** | Root path to resolve the include from | the file's path. |
 | **addDependencyTo** | Object with addDependency() method, taking file path as an argument. Called whenever a file is included. | |
+| **alias**| Object with alias mappings, { key: path } | |
```

Comment:
Could I get a slightly more clear description? People will be coming in who have no idea at all what this is 😁